src/components/sortableList/SortableListItem.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, {PropsWithChildren, useCallback, useContext} from 'react';
+import {LayoutChangeEvent} from 'react-native';
+import {
+  useSharedValue,
+  useAnimatedReaction,
+  withTiming,
+  Easing,
+  runOnJS,
+  useAnimatedStyle,
+  withSpring
+} from 'react-native-reanimated';
+import {GestureDetector, GestureUpdateEvent, PanGestureHandlerEventPayload} from 'react-native-gesture-handler';
+import View from '../view';
+import SortableListContext from './SortableListContext';
+import usePresenter from './usePresenter';
+import useDragAfterLongPressGesture from './useDragAfterLongPressGesture';
+
+interface SortableListItemProps {
+  index: number;
+}
+
+type Props = PropsWithChildren<SortableListItemProps>;
+
+const animationConfig = {
+  easing: Easing.inOut(Easing.ease),
+  duration: 350
+};
+
+const SortableListItem = (props: Props) => {
+  const {children, index} = props;
+
+  const {itemHeight, itemsOrder, onChange} = useContext(SortableListContext);
+  const {getTranslationByIndexChange, getItemIndexById, getIndexByPosition, getIdByItemIndex} = usePresenter();
+  const translateY = useSharedValue(0);
+  const tempTranslateY = useSharedValue(0);
+  const tempItemsOrder = useSharedValue(itemsOrder.value);
```

Comment:
Consider moving these to `useDragAfterLongPressGesture` hook. 
I'd move the logic to the hook, any reason onDragStart, onDragUpdate, onDragEnd shouldn't be part of the hook's code?

Reply:
Yes, very possible.
But I prefer to do it once the components are stable and we know for sure the logic shared between them is the same.

Reply:
So let's keep it as is? Do you want to add a task a TODO or...?

Reply:
Why not fix it? Is it a big change? 
If we'll decide not to share code, the code we'll just stuck there..